examples/stable_memory/test/test.ts

```diff
@@ -0,0 +1,78 @@
+import { cleanDeploy, run_tests, Test } from 'azle/test';
+import { createActor } from './dfx_generated/stable_memory';
+
+const stable_memory_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
+    agentOptions: {
+        host: 'http://127.0.0.1:8000'
+    }
+});
+
+const tests: Test[] = [
+    ...cleanDeploy('stable_memory'),
+    {
+        name: 'stable size',
+        test: async () => {
+            const result = await stable_memory_canister.stable_size();
+
+            return {
+                ok: result === 0
+            };
+        }
+    },
+    {
+        name: 'stable64 size',
+        test: async () => {
+            const result = await stable_memory_canister.stable64_size();
+
+            return {
+                ok: result === 0n
+            };
+        }
+    },
+    {
+        name: 'stable grow',
+        test: async () => {
+            const old_size = await stable_memory_canister.stable_size();
+            const new_pages = 5;
+            const result = await stable_memory_canister.stable_grow(new_pages);
+            const new_size = await stable_memory_canister.stable_size();
+
+            return {
+                ok:
+                    'ok' in result &&
+                    result.ok === old_size &&
+                    new_pages + old_size === new_size
+            };
+        }
+    },
+    {
+        name: 'stable grow to max',
+        test: async () => {
+            const old_size = await stable_memory_canister.stable_size();
+            const MAX_STABLE_MEM = 65536;
+            const new_pages = MAX_STABLE_MEM - old_size;
+            const result = await stable_memory_canister.stable_grow(new_pages);
+            const new_size = await stable_memory_canister.stable_size();
+
+            return {
+                ok:
+                    'ok' in result &&
+                    result.ok === old_size &&
+                    new_pages + old_size === new_size
+            };
+        }
+    },
+    {
+        name: 'stable grow out of memory',
```

Comment:
I'm not sure how big a deal this is, but this test is dependent upon the state of the last one. If we were using a more robust framework I would suggesting clearing the memory between each test or something and then doing all your setup for this test within this function. That way it won't break if we add another test or move this one above the "stable grow to max" test in the future.

Reply:
Do we have an easy way to clear the memory? The only way I know how to do it is to uninstall the canister code.

Reply:
@lastmjs said that we should be okay to do this for these tests since they are smoke tests. For the property tests we will want to be much more exact.